js/battle.js

```diff
@@ -4027,12 +4035,14 @@ var Battle = (function () {
 				var item = Tools.getItem(args[2]);
 				var effect = Tools.getEffect(kwargs.from);
 				var ofpoke = this.getPokemon(kwargs.of);
-				poke.item = '';
+				poke.item = item.name;
```

Comment:
``Pokemon#item`` is no longer accurate?!

We should look into another way to implement this... The existence of item history shouldn't imply that there is no item (history shouldn't be limited to removal but it should also include item transfer).